diode-server/dbstore/postgres/repository.go

```diff
@@ -0,0 +1,255 @@
+package postgres
```

Comment:
the insides of this don't have much in the way of coverage, we could consider adding something like `https://github.com/fergusstrange/embedded-postgres` to get more interesting tests of the system I think.

may not be worthwhile immediately especially if you're considering changing the interface.

Reply:
Agree, coverage dropped overall by ~5%, to be addressed separately 👍